src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/SqlUtil.cs

```diff
@@ -271,142 +496,193 @@ internal static Exception CannotGetDTCAddress()
         }
+
         static internal Exception SettingCredentialWithInteractiveArgument()
         {
             return ADP.Argument(StringsHelper.GetString(Strings.SQL_SettingCredentialWithInteractive));
         }
+
         static internal Exception SettingCredentialWithDeviceFlowArgument()
         {
             return ADP.Argument(StringsHelper.GetString(Strings.SQL_SettingCredentialWithDeviceFlow));
         }
+
         static internal Exception SettingCredentialWithNonInteractiveArgument(string authenticationMode)
         {
             return ADP.Argument(StringsHelper.GetString(Strings.SQL_SettingCredentialWithNonInteractive, authenticationMode));
         }
+
         static internal Exception SettingCredentialWithIntegratedInvalid()
         {
             return ADP.InvalidOperation(StringsHelper.GetString(Strings.SQL_SettingCredentialWithIntegrated));
         }
+
         static internal Exception SettingCredentialWithInteractiveInvalid()
         {
             return ADP.InvalidOperation(StringsHelper.GetString(Strings.SQL_SettingCredentialWithInteractive));
         }
+
         static internal Exception SettingCredentialWithDeviceFlowInvalid()
         {
             return ADP.InvalidOperation(StringsHelper.GetString(Strings.SQL_SettingCredentialWithDeviceFlow));
         }
+
         static internal Exception SettingCredentialWithNonInteractiveInvalid(string authenticationMode)
         {
             return ADP.InvalidOperation(StringsHelper.GetString(Strings.SQL_SettingCredentialWithNonInteractive, authenticationMode));
         }
-        internal static Exception NullEmptyTransactionName()
-        {
-            return ADP.Argument(StringsHelper.GetString(Strings.SQL_NullEmptyTransactionName));
-        }
-        internal static Exception UserInstanceFailoverNotCompatible()
-        {
-            return ADP.Argument(StringsHelper.GetString(Strings.SQL_UserInstanceFailoverNotCompatible));
-        }
-        internal static Exception CredentialsNotProvided(SqlAuthenticationMethod auth)
-        {
-            return ADP.InvalidOperation(StringsHelper.GetString(Strings.SQL_CredentialsNotProvided, DbConnectionStringBuilderUtil.AuthenticationTypeToString(auth)));
-        }
+
         internal static Exception ParsingErrorLibraryType(ParsingErrorState state, int libraryType)
         {
             return ADP.InvalidOperation(StringsHelper.GetString(Strings.SQL_ParsingErrorAuthLibraryType, ((int)state).ToString(CultureInfo.InvariantCulture), libraryType));
         }
+
         internal static Exception InvalidSQLServerVersionUnknown()
         {
             return ADP.DataAdapter(StringsHelper.GetString(Strings.SQL_InvalidSQLServerVersionUnknown));
         }
+
         internal static Exception SynchronousCallMayNotPend()
         {
             return new Exception(StringsHelper.GetString(Strings.Sql_InternalError));
         }
+
+#if NETCOREAPP
```

Comment:
Recent PR #2486 streamlined compilation symbols. You should pull from main and then update this.
```suggestion
#if NET6_0_OR_GREATER
```